132 bar chart for daily steps

NathanMoes opened the pull request:
Finished basics for the bar chart. Styling and database pulls are the only things not included

xuvea commented:
Works on my end. I would recommend adding a placeholder for data in the utils file to test your charts. What is the apple function for though? Might be something to remove unless it is being used moving forward. Besides that it looks good!

![image](https://user-images.githubusercontent.com/90873349/213941851-3e96597c-8c4d-43d7-9bc3-103327c92782.png)

NathanMoes commented:
> Works on my end. I would recommend adding a placeholder for data in the utils file to test your charts. What is the apple function for though? Might be something to remove unless it is being used moving forward. Besides that it looks good!
> 
> ![image](https://user-images.githubusercontent.com/90873349/213941851-3e96597c-8c4d-43d7-9bc3-103327c92782.png)
Apple function somehow broke it at times when I was testing it. I don't know why, but it should work withought it. Removed it just now and added some dummy data to utils
